faqs/_posts/2016-03-30-what-are-xls-reports.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: "What is an XLS Report?"
+permalink: /faq/what-is-an-xls-report
+category: faq
+---
+
+### What is an XLS Report?
+
+An XLS Report is an analytic tool available to Ona users that allows you to create custom reports in Excel™ using your data on Ona.  At the heart of XLS Reports is the flexible power of Excel.  Everything that’s possible in Excel, from formulas to pivot tables to charts, can be used in your XLS Reports.      
+
+XLS Reports are always updated with your form’s latest data and can be downloaded at any time from the form’s Overview page.  
+
+The sections below walk you through the steps of generating an XLS Report.  The [Mali Airports dataset](https://ona.io/acme/2280/18477) is used as an example.
+
+### How do I create an XLS Report Template? 
+
+An XLS Report Template is the Excel file that is used to generate an XLS Report.  An XLS Report Template contains the formulas, macros, and formatting that you want included in your final XLS Report.  Once the XLS Report Template is uploaded to your form, Ona sends the template and the data to [j2x.ona.io](https://j2x.ona.io/) which then sends back a downloadable report.
+
+Here is the [XLS Report Template](https://docs.google.com/a/ona.io/spreadsheets/d/1bfeYJOWt72NBP_nZYruJOgEVIUvIgPTo2av4bjw67X4/edit#gid=965777567) created for the Mali Airports dataset.  You can use it to follow the instructions below.
+
+### Tags in an XLS Report Template
+
+In order to generate a report based on specific parameters, you must indicate on the template what information is to be extracted from the data set. You will use jXLS tags to pull in the raw data.
+
+<br>
+![](/content/screenshots/faq/xls-reports-1.png)
+
+A.  `</jx:forEach items= "${data}" var=”airport”>`is called the opening tag. In your XLS Report Template, replace "airport" with a name that is relevant to your dataset.
+
+After naming the dataset in the opening tag, you then order the dataset based on parameters collected in the form. 
+In the example above, we have ordered the “airport” dataset based on region, municipality, code, name, type, scheduled service and elevation parameters, which correspond to specific data fields in the form. 
+
+B.  `<jx:forEach>` is known as the closing tag. It invokes the end of the dataset.
+
+C.  `${airport.iso_region}` tag searches for region from ‘airport data’ and puts it into the corresponding cell.
```

Comment:
Why is airpot data in quotes?  Is this the form id?  This needs to be explained or simply say it searches for region from the airport dataset.  Also, I would change the field names in the xlsform to match this better.  I'm guessing in the form, the field name is iso_region, so it's confusing to reference both iso_region and region in the same line.